src/locales/ru/translation.json

```diff
@@ -4,12 +4,14 @@
     "successfully_signed_in": "Успешный вход"
   },
   "home": {
-    "featured": "Рекомендованное",
+    "featured": "Рекомендации",
     "surprise_me": "Удиви меня",
     "no_results": "Ничего не найдено",
     "hot": "Сейчас в топе",
     "start_typing": "Начинаю вводить текст для поиска...",
-    "weekly": "📅 Лучшие игры недели"
+    "weekly": "📅 Лучшие игры недели",
+    "achievements": "🏆 Игры, в которых нужно победить"
+
```

Comment:
could you just remove this empty line?
this is probably the reason why lint threw an error